src/Uno.UWP/Devices/Power/BatteryReport.Android.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.Devices.Power
+{
+	public partial class BatteryReport
+	{
+		public int? ChargeRateInMilliwatts;
+		public int? DesignCapacityInMilliwattHours;
+		public int? FullChargeCapacityInMilliwattHours;
+		public int? RemainingCapacityInMilliwattHours;
+		public Windows.System.Power.BatteryStatus Status;
+
+		private Windows.System.Power.BatteryStatus UWPstatusFromAndroStatus(int status)
+		{
+			switch (status)
+			{
+				case 1: // BATTERY_STATUS_UNKNOWN
+					return System.Power.BatteryStatus.Idle;
+				case 2: // BATTERY_STATUS_CHARGING
+					return System.Power.BatteryStatus.Charging;
+				case 3: // BATTERY_STATUS_DISCHARGING
+					return System.Power.BatteryStatus.Discharging;
+				case 4: // BATTERY_STATUS_NOT_CHARGING
+					return System.Power.BatteryStatus.Idle;
+				case 5: // BATTERY_STATUS_FULL
+					return System.Power.BatteryStatus.Idle;
```

Comment:
```suggestion
				case 1: // BATTERY_STATUS_UNKNOWN
					return BatteryStatus.Idle;
				case 2: // BATTERY_STATUS_CHARGING
					return BatteryStatus.Charging;
				case 3: // BATTERY_STATUS_DISCHARGING
					return BatteryStatus.Discharging;
				case 4: // BATTERY_STATUS_NOT_CHARGING
					return BatteryStatus.Idle;
				case 5: // BATTERY_STATUS_FULL
					return BatteryStatus.Idle;
```

Reply:
BatteryStatus is not known in this context, so it has to be prefixed with namespace.